docs/core/config/auth/passdb.md

```diff
@@ -442,6 +442,18 @@ a load spike of everybody getting logged in at exactly the same time.
 
 Do not perform any authentication, just store extra fields if user is found.
 
+##### `client_fp`
+
+Match client certificate or public key fingerprint. See [[setting,ssl_peer_certificate_fingerprint_hash]].
```

Comment:
These could use a better explanation of what exactly they do. What happens when it matches? What happens it doesn't match (I think the whole auth fails, not just the one passdb)?

Reply:
Also explanation how it interacts with `auth_ssl_require_client_cert` setting.